src/develop/imageop_math.h

```diff
@@ -99,21 +99,8 @@ void dt_iop_clip_and_zoom_8(const uint8_t *i, int32_t ix, int32_t iy, int32_t iw
                             int32_t ibh, uint8_t *o, int32_t ox, int32_t oy, int32_t ow, int32_t oh,
                             int32_t obw, int32_t obh);
 
-void dt_iop_YCbCr_to_RGB(const dt_aligned_pixel_t yuv, dt_aligned_pixel_t rgb);
```

Comment:
I'm not sure we want to remove unused functions from a generic API like this. It is like striping down some part of an API. It may well be needed at some point in the future.